frontend/apps/docs/content/docs/index.mdx

```diff
@@ -1,44 +1,68 @@
 ---
-title: Getting Started
-description: Getting Started with Liam ERD
+title: Welcome to Liam ERD
+description: Liam ERD is a tool that effortlessly generates beautiful and easy-to-read ER diagrams.
 ---
 
 import { Package, Globe } from "lucide-react";
 
-Welcome to **Liam ERD**, the open-source tool designed to simplify database design and visualization. Whether you're working on a public open-source project or managing private repositories, Liam ERD makes it easy to generate and share **beautiful, interactive ER diagrams**.
-
 ![Liam logo](/images/default.png)
 
+Liam ERD is an open-source tool that instantly generates beautiful, interactive ER diagrams from your database. Whether you’re working on public or private repositories, Liam ERD helps you visualize complex schemas with ease.
+
 ## Why Choose Liam ERD?
 
-Liam ERD stands out with its:
+- **Beautiful UI & Interactive**: A clean design and intuitive features (like panning, zooming, and filtering) make it easy to understand even the most complex databases.
+- **Simple Reverse Engineering**: Seamlessly turn your existing database schemas into clear, readable diagrams.
+- **Effortless Setup**: Get started with zero configuration—just provide your schema, and you’re good to go.
+- **High Performance**: Optimized for both small and large projects, easily handling 100+ tables.
+- **Fully Open-Source**: Contribute to the project and shape Liam ERD to fit your needs.
+
+## Explore Diagrams
+
+Curious about what Liam ERD can do? Visit https://liambx.com/erd/galaxy to browse a variety of example ER diagrams from real-world schemas.
+
+## Supported Schemas
+
+Legend:
+
+- ✅: Supported
+- ❌: Not in progress
```

Comment:
[nits]
❌ looks like we are not going to support this.
⛔ is more suitable? I'm not sure native sense of emoji 😅

Reply:
Looks great!! ⛔

Reply:
fixed: https://github.com/liam-hq/liam/pull/366/commits/5bc68d5c16bbff9faf969ac549a8c8e1f15f4548